doc/stdlib/collections.rst

```diff
@@ -221,6 +221,12 @@ Iterable Collections
 
    Returns an array containing only the unique elements of the iterable ``itr``\ , in the order that the first of each set of equivalent elements originally appears. If ``dim`` is specified, returns unique regions of the array ``itr`` along ``dim``\ .
 
+   .. code-block:: julia
+
+       unique(f, itr)
+
+   Returns an array containing one value from ``itr`` for each unique value produced by ``f`` applied to elements of ``itr``\ .
+
```

Comment:
I tried, but I've hit my non-deterministic error rate-limit for the day. The system refuses to recognize the function signatures. (yes, I rebuilt Julia, matched the signatures in helpdb and collections.rst, rebuilt julia again, ...).

Also, every 3rd or 4th run of genstdlib I get:

```
ERROR: LoadError: UnicodeError: invalid character index
 [inlined code] from ./int.jl:187
```